### PR TITLE
Update events.json

### PR DESCRIPTION
**Pull request by kacycrider**

### CODE REVIEW

File: app/config/tstvhq/events.json [hunk 1]
```diff
@@ -1,59 +1,59 @@
...
-            "description": "Serpentinae DJ Set.",
-            "room": "kitchen",
-            "start": "2020-12-26T23:00:00-05:00",
-            "end": "2020-12-27T01:00:00-05:00",
+        {
+            "name": "Laughing in Disbelief: Live Interview of Ada King, Director of TSTVHQ",
+            "description": "Jack Matirko interviews director of TSTVHQ Ada King with host Andy Hall*,
+            "room": "infernalTheater",
+            "start": "2020-12-27T22:30:00-05:00",
+            "end": "2020-12-27T23:00:00-05:00",
```
Comment (hndrewaall) [inline app/config/tstvhq/events.json:32]: missing a quote here

**Review by hndrewaall — CHANGES_REQUESTED** [review]

File: app/config/tstvhq/events.json [hunk 2]
```diff
@@ -1,59 +1,59 @@
...
         },
-         {
-            "name": "Shiva Honey DJ Set",
-            "description": "Serpentinae DJ Set.",
-            "room": "kitchen",
-            "start": "2020-12-26T23:00:00-05:00",
-            "end": "2020-12-27T01:00:00-05:00",
+        {
+            "name": "Laughing in Disbelief: Live Interview of Ada King, Director of TSTVHQ",
+            "description": "Jack Matirko interviews director of TSTVHQ Ada King with host Andy Hall*,
```
Comment (hndrewaall) [inline app/config/tstvhq/events.json:29]: and here

File: app/config/tstvhq/events.json (hunk 1 above)
Comment (kacycrider) [reply app/config/tstvhq/events.json:32]: butts

Comment (kacycrider) [reply app/config/tstvhq/events.json:32]: im not seeing this one

**Review by hndrewaall — APPROVED** [review]

**Pull request merged**